src/codecs/Reaction.ts

```diff
@@ -0,0 +1,60 @@
+import { ContentTypeId, ContentCodec, EncodedContent } from '../MessageContent'
+
+// xmtp.org/text
+//
+// This content type is used for a plain text content represented by a simple string
+export const ContentTypeReaction = new ContentTypeId({
+  authorityId: 'xmtp.org',
+  typeId: 'reaction',
+  versionMajor: 1,
+  versionMinor: 0,
+})
+
+export type Reaction = {
+  /**
+   * The message ID for the message that is being reacted to
+   */
+  reference: string
+  /**
+   * The action of the reaction
+   */
+  action: 'added' | 'removed'
+  /**
+   * The content of the reaction
+   */
+  content: string
+}
+
+export type ReactionParameters = Pick<Reaction, 'action' | 'reference'> & {
+  encoding: 'UTF-8'
```

Comment:
i hardcoded this type to eliminate the need for an enum. we can add more in the future if necessary.